src/parser.rs

```diff
@@ -1793,18 +1795,12 @@ impl Parser {
                     // Ignore the error and back up to where we were before.
                     // Either we'll be able to parse a valid nested join, or
```

Comment:
This whole comment should be reworded (as well as the bit in ast.rs I linked to earlier), as otherwise it becomes self-contradictory. I guess I'll do it myself later.

Reply:
Ok , i didn't touched  this comment - and according to my english skills i don't sure that I could improve it :) .